README.md

```diff
@@ -50,8 +50,14 @@ public final class MyClassWithTracing {
   private static final Tracer tracer = Tracing.getTracer();
 
   public static void doWork() {
-    // Create a child Span of the current Span.
-    try (Scope ss = tracer.spanBuilder("MyChildWorkSpan").startScopedSpan()) {
+    // Create a child Span of the current Span. Always record events for this span and force it to 
+    // be sampled. This makes it easier to try out the example, but unless you have a clear use 
+    // case, you don't need to explicitly set record events or sampler.
+    try (Scope ss = 
+         tracer.spanBuilder("MyChildWorkSpan")
+           .setRecordEvents(true)
```

Comment:
maybe we can special-case and when sampler is always, imply event recording? Seems a bit cluttery and the intent is probably clear. Otherwise, maybe we just need to be more clear about when you'd *no* record events, yet always sample.

Reply:
/cc @bogdandrutu